### PR TITLE
Add ACs for Gov Transfer cancellation proposal/network history restore/split permitted combos into ACs

### PR DESCRIPTION
**Pull request by vega-paul**

### CODE REVIEW

File: protocol/0028-GOVE-governance.md [hunk 1]
```diff
@@ -617,3 +617,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 ##### Cancelling governance transfers
 
 - Only recurring governance transfers can be cancelled via governance cancel transfer proposal. Trying to cancel any other transfer should fail upon validation of the proposal.(<a name="0028-GOVE-107" href="#0028-GOVE-107">0028-GOVE-107</a>)
+- Cancellation of a governance transfer can only be done by the proposer of the original proposal to create the recurring transfer (<a name="0028-GOVE-108" href="#0028-GOVE-108">0028-GOVE-108</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]: This is completely at odds with what's in the spec above:

```
This is done as a governance proposal. Takes a transfer ID (which is the proposal ID of the original transfer) and would cancel a recurring governance transfer. Only recurring governance transfers can be cancelled via governance cancel transfer proposal. Trying to cancel any other transfer should fail upon validation of the proposal.
```

File: protocol/0028-GOVE-governance.md [hunk 2]
```diff
@@ -617,3 +617,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
...
+- No cancellation transactions/events wil be produced after the block/seq which contained the cancellation transaction (<a name="0028-GOVE-109" href="#0028-GOVE-109">0028-GOVE-109</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]: There is no cancellation transaction; there has to be a proposal / vote. Please make this more precise.

File: protocol/0028-GOVE-governance.md [hunk 3]
```diff
@@ -617,3 +617,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
...
+- Attempts to cancel a recurring transfer which has ended will result in a transaction rejection which states the transfer does not exist (<a name="0028-GOVE-110" href="#0028-GOVE-110">0028-GOVE-110</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]: Again, more precise wording needed.

File: protocol/0028-GOVE-governance.md [hunk 4]
```diff
@@ -617,3 +617,7 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
...
+- A public key which attempts to cancel a recurring transfer which they did not propose will result in a transaction rejection which states the transfer cannot be cancelled by the party (<a name="0028-GOVE-111" href="#0028-GOVE-111">0028-GOVE-111</a>)
```
Comment (davidsiska-vega) [inline protocol/0028-GOVE-governance.md]: It's not about the key which proposed it; any can submit a proposal to cancel a governance transfer.

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
This needs complete rewrite. Please look at the spec at what is says about cancelling governance transfers.

File: protocol/0028-GOVE-governance.md (hunk 1 above)
Comment (vega-paul) [reply protocol/0028-GOVE-governance.md]: sorry about that @davidsiska-vega can you please review again?

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**